packages/features/portfolio/src/containers/portfolio-overview/portfolio-overview.tsx

```diff
@@ -65,7 +65,7 @@ const PortfolioOverview = () => {
       ? SECONDS_IN_DAY * 2 * 15 * 3
       : slot === '1y'
       ? SECONDS_IN_DAY * 365
-      : undefined
```

Comment:
I made this change for the "All" selector so it should be undefined. When it is undefined it will auto-detect the earliest date.